README.md

```diff
@@ -51,3 +51,16 @@ using the following command:
 ```shell
 $ ENABLE_WEBHOOKS=false make run install
 ```
+
+## Metrics
+
+Apart from the [metrics provided by controller-runtime](https://book.kubebuilder.io/reference/metrics-reference.html)
+by default, this operator exports the following custom metrics:
+
+| Metric name                      | Type      | Description                                                                |
+|----------------------------------|-----------|----------------------------------------------------------------------------|
+| release_attempt_concurrent_total | Gauge     | Total number of concurrent releases.                                       |
+| release_attempt_duration_seconds | Histogram | Total duration of the release from the moment the PipelineRun was created. |
+| release_attempt_invalid_total    | Counter   | Number of invalid releases.                                                |
+| release_attempt_running_seconds  | Histogram | Time for a release to change its state to running.                         |
```

Comment:
The description should be updated to be more clear

Reply:
Changed to `Time for releases to change their state to running`. Not sure it's even better. What do you think?

Reply:
I like it better, but maybe like (and you'll probably hate me for suggesting the word total now after complaining about it before) `Total time for all releases to change their state to running from created`